test/cppsuite/README.md

```diff
@@ -1 +1,124 @@
+
+| Phase    | Description |
+| -------- | ----------- |
+| Load     | Initializes the oldest and stable lags using the values retrieved from the configuration file. |
+| Run      | Updates the oldest and stable lags as well as the oldest and stable timestamps. |
+| Finish   | N/A |
+
+## Metrics Monitor
+The metrics monitor polls database metrics during and after test execution. If a metric is found to be outside the acceptable range defined in the configuration file then the test will fail. When a test ends, the metrics monitor can export the value of these statistics to a JSON file that can be uploaded to Evergreen or Atlas.
+
+| Phase    | Description |
+| -------- | ----------- |
+| Load     | Retrieves the metrics from the configuration file that need to be monitored during and after the test. |
+| Run      | Checks runtime metrics and fails the test if one is found outside the range defined in the configuration file. |
+| Finish   | Checks post run metrics and fails the test if one is found outside the range defined in the configuration file. Exports any required metrics to a JSON file if the test is successful. |
+
+All the components and their implementation can be found in the [component](https://github.com/wiredtiger/wiredtiger/tree/develop/test/cppsuite/src/component) folder.
+
+# Test configuration file
+A test configuration is a text file that is associated with a test and defines the workload. The test configuration file contains top-level settings such as the test duration or cache size and component level settings that define their behavior. The format is the following:
+
+```cpp
+# Top level settings
+duration_seconds=15,
+cache_size_mb=200,
+...,
+# Components settings
+timestamp_manager=
+(
+  enabled=true,
+  oldest_lag=1,
+  stable_lag=1,
+  ...,
+),
+...,
+```
+
+Each test has a default configuration file which is the test name followed by the suffix `_default` and the extension `.txt`. It is possible to use any configuration file when executing a test, see [here](how_to_use_cppsuite.md) for more details.
+
+All the different configurable items are defined in [test_data.py](https://github.com/wiredtiger/wiredtiger/blob/develop/dist/test_data.py).
+
+
+# How to execute a test
+A tutorial is available [here](how_to_use_cppsuite.md).
```

Comment:
It would be worth double checking the hyperlinks work once both this ticket and WT-9111 have been merged.

Reply:
I've merged the branches locally and checked the links in both docs. We've got 1 link in the README to update and the rest are working correctly.